.gitmodules

```diff
@@ -1,36 +1,42 @@
-[submodule "cosim/import/black-parrot"]
-	path = cosim/import/black-parrot
-	url = https://github.com/black-parrot/black-parrot.git
+[submodule "import/black-parrot"]
+    path = import/black-parrot
+    url = https://github.com/black-parrot/black-parrot.git
     branch = dev
     ignore = dirty
-[submodule "cosim/import/black-parrot-tools"]
-	path = cosim/import/black-parrot-tools
-	url = https://github.com/black-parrot/black-parrot-tools
+[submodule "import/black-parrot-tools"]
+    path = import/black-parrot-tools
+    url = https://github.com/black-parrot/black-parrot-tools
     branch = master
     ignore = dirty
-[submodule "cosim/import/black-parrot-subsystems"]
-	path = cosim/import/black-parrot-subsystems
+[submodule "import/black-parrot-subsystems"]
+    path = import/black-parrot-subsystems
     url = https://github.com/black-parrot-hdk/black-parrot-subsystems
     branch = main
     ignore = dirty
-[submodule "software/import/black-parrot-sdk"]
-	path = software/import/black-parrot-sdk
-	url = https://github.com/black-parrot-sdk/black-parrot-sdk
+[submodule "import/bsg_manycore"]
+    path = import/bsg_manycore
+    url = https://github.com/bespoke-silicon-group/bsg_manycore
     branch = master
     ignore = dirty
-[submodule "cosim/import/bsg_manycore"]
-	path = cosim/import/bsg_manycore
-	url = https://github.com/bespoke-silicon-group/bsg_manycore
+[submodule "import/basejump_stl"]
+    path = import/basejump_stl
+    url = https://github.com/bespoke-silicon-group/basejump_stl
     branch = master
     ignore = dirty
-[submodule "cosim/import/basejump_stl"]
-	path = cosim/import/basejump_stl
-	url = https://github.com/bespoke-silicon-group/basejump_stl
+
+[submodule "import/black-parrot-sdk"]
+    path = import/black-parrot-sdk
+    url = https://github.com/black-parrot-sdk/black-parrot-sdk
     branch = master
     ignore = dirty
-[submodule "software/import/bsg_replicant"]
-	path = software/import/bsg_replicant
-	url = https://github.com/bespoke-silicon-group/bsg_replicant
+[submodule "import/bsg_replicant"]
+    path = import/bsg_replicant
+    url = https://github.com/bespoke-silicon-group/bsg_replicant
+    branch = main
+    ignore = dirty
+[submodule "import/verilator"]
+    path = import/verilator
+    url = https://github.com/verilator/verilator
```

Comment:
What is the reason for a separate Verilator submodule here rather than using the one in black-parrot-tools?